src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1083,6 +1083,15 @@
   <data name="DoNotCreateTasksWithoutPassingATaskSchedulerMessage" xml:space="preserve">
     <value>Do not create tasks without passing a TaskScheduler</value>
   </data>
+  <data name="DoNotAddFinalizerToMemoryManagedTDerivedTypesDescription" xml:space="preserve">
+    <value>Adding a finalizer to a MemoryManager&lt;T&gt; derived type is likely an indication of a bug, as it suggests a native resource that could have been handed out in a Span&lt;T&gt; is getting cleaned up and potentially while it’s still in use by the Span&lt;T&gt;.</value>
```

Comment:
Just copied from the issue ...

Reply:
It'll probably get some PM tweaking by Manish or Immo.  It's mostly fine as-is, though I would simplify it to

`Adding a finalizer to a MemoryManager<T>-derived type creates the possibility that the memory will be freed while it is still in use by a Span<T>.`

For both the message and the description.

Reply:
Sure, will tag Immo on the PR then

Reply:
Actually i read the email that we are going to have API review for each analyzer, i guess that would discuss the wording too